src/net/util/show_packet.h

```diff
@@ -20,7 +20,7 @@ static inline void show_packet(uint8_t *raw, int size, char *title) {
 	if (!&mod_logger)
 		return;
 
-	if (&mod_logger.logging &&
+	if (//&mod_logger.logging &&
```

Comment:
Please consider also `logging_raw` function, which called like
```
#define log_logp(level, fmt, ...) \
        if (&mod_logger) \
                logging_raw(&mod_logger.logging, level, "%s: %s: " fmt "\n", \
```
and defined to be 
```
void logging_raw(struct logging *logging, int level, const char* fmt, ...) {
        if (logging
```

I find this definition and the redundant check at least consistent. 
In case the check deleted `logging_raw` should be altered similar way, I suppose